### PR TITLE
add calc cmd 

### PR DESCRIPTION
**Pull request by ademyrodev**
not using any library, so there is a 'calculator' class in calc.py. The command definition itself is at line 138.

Example use:
``v!calc 1 + 9``
Result: ``10``
``v!calc 1 + 2 * 3``
Result: ``7``
``v!calc (1 + 2) * 3``
Result: ``9``
``v!calc 29 + 8(9/2)``
Result: ``65``

### CODE REVIEW

File: bot/commands/calc.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            self.advance()
+            return expr
+        
+        self.error()
+            
+
+class cmd(Command):
+    # The command.
+    name = "calc"
+    usage = "<*expression>"
```
Comment (logicguy1) [inline bot/commands/calc.py:141]: Your usage text should say `commandname arguments`, you are missing command name

File: bot/commands/calc.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    description = "Calculates the given expression and sends the result."
```
Comment (logicguy1) [inline bot/commands/calc.py:142]: Is it possible to include a list of valid operators in the description?

File: bot/commands/calc.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        # Create a calculator instance.
+        calculator = Calculator(expression)
+        
+        # Get the result.
+        result = calculator.expr()
+        
+        # Send the result:
+        await message.channel.send(f"_{expression}_ = **{result:g}**")
+        
+        # Nice.
```
Comment (logicguy1) [inline bot/commands/calc.py:160]: Nice

**Review by logicguy1 — COMMENTED** [review]
Just a few things to look at :p

File: bot/commands/calc.py (hunk 1 above)
Comment (ademyrodev) [reply bot/commands/calc.py:141]: Will replace it with ``calc <*expression>``.

File: bot/commands/calc.py (hunk 2 above)
Comment (ademyrodev) [reply bot/commands/calc.py:142]: Sure! Will do.

**Comment by ademyrodev** [comment]
Just updated it.

**Pull request merged**